### ISSUE TITLE
some questions on IDnet

### ISSUE THREAD

**Issue by lijie-vipl-ict** [ShuangLI59/person_search#79]
Hi @Cysu and @ShuangLI59 :+1: 
    Recently,i have a work also about Re-ID,but I came across a very tricky question  which you may have encountered.So,i want ask u a few question:
    1.If i don't have unlabed identities , on my own dataset,i just have labeled identities,can i trainning the network on my dataset. Can I set the length of CQ to 0?
    2. Now there is such a situation，I  use your method trainning the network  on my own dataset ,when i testing the model,i find the accuracy of testset is very low (about 10% top1),but the acc of the trainset is very high,does it overfitting?or , i ignored any important points? Do u have any advices?
   3.  In your paper ,you have mentioned "each image only contains a few identities",and your LUT and CQ are using to sovle this  batch problem(softmax),right? Have you tried such a method like this: in the foward process , store the different identities feature,and randomly select some features, belong to different classes ,to train the softmax?  Is there any problem with the gradient of this method?

I'm looking foward your answer ,in a really big hurry,and thank you!!

**Comment by Cysu** [comment]
1. It might work but I'm not sure about it. It would be better to modify the code and net architecture accordingly, rather than directly set length of CQ to 0.
2. It could be overfitting. But my experience is that when something is unreasonable, there are probably some bugs / glitches in the code. I would recommend visualizing the training / test inputs and results first.
3. We only did some experiments that randomly subsample from the CQ of unlabeled identities, as illustrated in the paper. I think subsampling labeled identities will also work.
